src/routes/api/configured/api-configured-get.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+import { getLogger } from "config/logger";
+import { getConfiguredAppProperties } from "utils/app-properties-utils";
+import { Subscription } from "models/subscription";
+
+/**
+ * Makes a request to Jira to get jiraHosts is-configured app property
+ */
+export const ApiConfiguredGet = async (req: Request, res: Response): Promise<void> => {
```

Comment:
Are we using this endpoint anywhere? Where do we need this?

Reply:
Only using it manually to verify. Happy to pull it out but treat it like a feature flag. 
If we have everything deployed(the other storys for displaying things) otherwise its just handy to validate that the script worked.